frontend/fingertips-frontend/components/organisms/EnglandAreaTypeTable/EnglandAreaTypeTable.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { render, screen } from '@testing-library/react';
+import {
+  EnglandAreaTypeIndicatorData,
+  EnglandAreaTypeTable,
+} from '@/components/organisms/EnglandAreaTypeTable/index';
+import {
+  HealthDataForArea,
+  HealthDataPointTrendEnum,
+} from '@/generated-sources/ft-api-client';
+import { areaCodeForEngland } from '@/lib/chartHelpers/constants';
+import { allAgesAge, noDeprivation, personsSex } from '@/lib/mocks';
+
+const mockEnglandHealthData: HealthDataForArea = {
+  areaCode: areaCodeForEngland,
+  areaName: 'England',
+  healthData: [
+    {
+      year: 2008,
+      count: 222,
+      value: 890.305692,
+      lowerCi: 441.69151,
+      upperCi: 578.32766,
+      ageBand: allAgesAge,
+      sex: personsSex,
+      trend: HealthDataPointTrendEnum.NotYetCalculated,
+      deprivation: noDeprivation,
+    },
+  ],
+};
+
+const mockNoHealthData: HealthDataForArea = {
+  areaCode: areaCodeForEngland,
+  areaName: 'England',
+  healthData: [],
+};
+
+const mockIndicatorData: EnglandAreaTypeIndicatorData[] = [
+  {
+    indicatorId: 1,
+    indicatorName: ' ',
+    period: '2008',
+    latestEnglandHealthData: mockEnglandHealthData.healthData[0],
+    unitLabel: '',
+  },
+  {
+    indicatorId: 2,
+    indicatorName: 'no data indicator',
+    period: '',
+    latestEnglandHealthData: mockNoHealthData.healthData[0],
```

Comment:
I think this could just be:
```suggestion
    latestEnglandHealthData: undefined,
```

Then you wouldn't need `mockNoHealthData`.